cpp/include/cugraph/graph_functions.hpp

```diff
@@ -448,11 +448,9 @@ void relabel(raft::handle_t const& handle,
  * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
  * handles to various CUDA libraries) to run graph algorithms.
  * @param graph_view Graph view object, we extract induced subgraphs from @p graph_view.
- * @param subgraph_offsets Pointer to subgraph vertex offsets (size == @p num_subgraphs + 1).
- * @param subgraph_vertices Pointer to subgraph vertices (size == @p subgraph_offsets[@p
- * num_subgraphs]). The elements of @p subgraph_vertices for each subgraph should be sorted in
- * ascending order and unique.
- * @param num_subgraphs Number of induced subgraphs to extract.
+ * @param subgraph_offsets Span pointing to subgraph vertex offsets
```

Comment:
Shouldn't we better be little more specific here?

Maybe add something like the following.
```
@p subgraph_offsets and @p subgraph_vertices provide vertex sets (or local vertex sets in multi-GPU) for @p subgraph_offsets.size() - 1 subgraphs to extract.

For the i'th subgraph to extract, one can extract the (local-)vertex set by accessing a subset of @p subgraph_vertices, where the range of the subset is [@p subgraph_offsetes[i], @p subgraph_offsets[i + 1]).

In multi-GPU, the vertex set for each subgraph is distributed in multiple-GPUs and each GPU holds only the vertices that are local to the GPU.
```

Reply:
Added the suggested text.